src/app/app.module.ts

```diff
@@ -1,16 +1,22 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { HttpClientModule } from '@angular/common/http';
 
```

Comment:
is that empty space on purpose ? :)

Reply:
Is it bad idea to visually separate external modules imports (such as @angular) from internal imports (I mean importing from files which are created within the project)?

Reply:
its good idea :) i was just wonder if that was an accident
lets follow this pattern in this project,
 we dont use it in a project tho